frontend/src/lib/utils/eventUsageLogic.ts

```diff
@@ -996,6 +996,7 @@ export const eventUsageLogic = kea<eventUsageLogicType>([
                 name: experiment.name,
                 id: experiment.id,
                 type: experiment.type,
+                // TODO fix these
```

Comment:
@annikaschmid @jurajmajerik @andehen What properties would you like to see on the `experiment created`, `experiment viewed`, and `experiment launched` events?

`filters: sanitizeFilterParams(experiment.filters)` returns a whole mess:

https://github.com/PostHog/posthog/blob/d15d062b798d4d5bf73928b5a2f05fc66d232d7b/frontend/src/lib/utils/eventUsageLogic.ts#L214-L237

Given no one noticed these were missing, I would just assume we drop it all and start from scratch.

Reply:
* How many metrics
* Types of metrics (funnel vs. trends), also per primary vs. secondary (see what use of secondary metrics are)
* Type of metric for the first metric
* # of funnel steps
* Use of events vs. actions vs. data warehouse
* Experiment calculated and actual durations (on the viewed and experiment stopped event)
* # of shared metrics applied
* Filter internal test accounts